src/components/RegisterForm.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import {TextField, Typography} from '@mui/material';
+import Button from '@mui/material/Button';
+import { Link } from 'react-router-dom';
+
+function RegisterForm() {
+  return (
+    <div className="login">
+      <Box
+        component="form"
+        sx={{
+          '& .MuiTextField-root': { m: 1, width: '25ch' },
+          border: '1px dashed grey',
+        }}
+        noValidate
+        autoComplete="off"
+      >
+        <Typography textAlign="center" variant="h4" gutterBottom>
+          Sign Up
+        </Typography>
+        <Typography textAlign="center" variant="subtitle1" gutterBottom>
+          It's free and only takes a minute
```

Comment:
So, this is an ESlinter error which you can solve by escaping the `'` sign like this:

`It&apos;s free and only takes a minute`

That is because of these kinds of characters and `&` for example. Can be harmless code, meaning that this can be used to harm the page look on the HTML code. And Also, to prevent XSS attacks.